Classes/Emogrifier.php

```diff
@@ -911,13 +911,14 @@ private function attributeValueIsImportant($attributeValue)
      *
      * @return void
      */
-    private function copyCssWithMediaToStyleNode(\DOMDocument $xmlDocument, \DOMXPath $xPath, $css)
+    private function copyCssWithMediaToStyleNode(\DOMDocument $xmlDocument, \DOMXPath $xPath, $css, $imports)
     {
-        if ($css === '') {
+        if ($css === '' && $imports === '') {
             return;
         }
 
         $mediaQueriesRelevantForDocument = [];
+        !$imports ?: $mediaQueriesRelevantForDocument[] = "\n".$imports."\n";
```

Comment:
Please don't use the Elvis operator for commands. (It should only be used for return values or assignments). Please use an IF instead.